src/argilla/client/feedback/dataset.py

```diff
@@ -777,13 +782,15 @@ def format_as(self, format: Literal["datasets"]) -> "Dataset":
                     value = Value(dtype="string")
                 elif question.settings["type"] == "rating":
                     value = Value(dtype="int32")
-                elif question.settings["type"] == "multi_label_selection":
+                elif question.settings["type"] == "ranking":
+                    value = Sequence({"rank": Value(dtype="int32"), "value": Value(dtype="string")})
```

Comment:
maybe the `rank` can be an `int16` or even `int8`

Reply:
Agree, but we should use an unsigned type instead, as ranks are from 1 to N, we can set it to `uint8` 👍🏻

Reply:
```suggestion
                    value = Sequence({"rank": Value(dtype="uint8"), "value": Value(dtype="string")})
```